app/managers.py

```diff
@@ -0,0 +1,26 @@
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.connect = sqlite3.connect("../cinema.sqlite")
```

Comment:
You can store table name as an attribute and then use it in the sql, to avoid hardcoding

Reply:
This hasn't been fixed, please parametrise database name